corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -393,7 +424,7 @@ def test_case_list_registration_form_advanced(self):
         case_module.case_type = 'dugong'
         update_form = app.new_form(1, 'Update Case', lang='en')
         update_form.unique_id = 'update_case_form'
-        update_form.actions.load_update_cases.append(LoadUpdateAction(
+        update_form.actions.load_update_cases.append(LoadUpdateAction(  # <-- e.g.
```

Comment:
True. Currently working on two tests for advanced forms and that comment is gone. (If I'd noticed that comment earlier, I wouldn't have committed it.)